packages/landing/src/components/debug.tsx

```diff
@@ -33,10 +33,42 @@ export class Debug extends Component<{ map: maplibre.Map }> {
           <label className="debug__label">Purple</label>
           <input type="checkbox" onClick={this.togglePurple} />
         </div>
+        {this.renderSourceToggle()}
       </div>
     );
   }
 
+  renderSourceToggle(): ComponentChild {
+    // TODO this is a nasty hack to detect if a direct imageryId is being viewed
+    if (!Config.map.layerId.startsWith('01')) return null;
```

Comment:
Might be good to have boolean property in `Config.map` to catch the `im_` as imagery?